### DISCUSSION TITLE
Running `just fmt` fails

### DISCUSSION THREAD

**Discussion by Adda0** [juspay/omnix#348] [Q&A] [answered]
```sh
 $ just fmt
treefmt
Error: failed to find treefmt config file: could not find [treefmt.toml .treefmt.toml] in <path>/omnix
error: Recipe `fmt` failed on line 9 with exit code 1
```
`treefmt` is not added as a dev shell dependency.

When `treefmt` is manually added through `nix shell`, for example:
```sh
$ just fmt
treefmt
Error: failed to find treefmt config file: could not find [treefmt.toml .treefmt.toml] in <path>/omnix
error: Recipe `fmt` failed on line 9 with exit code 1
```

**Comment by srid** [comment, marked as answer]
You should run it from the `nix develop` shell. Alternatively, setup `direnv` as README recommends:

<img width="282" alt="image" src="https://github.com/user-attachments/assets/10fab6d4-37b1-4d2f-a33a-8eeeae5a084f">

**Reply by Adda0** [reply]
Sorry. I must have somehow exited the dev shell by mistake before running the command. It works in the dev shell.
